fix: add missing trim() in innerText checks

rajatbaranwal opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [ ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or Gitpod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

<!-- Feel free to add any additional description of changes below this line -->

rajatbaranwal committed:
fix: add missing trim() in innerText checks

ilenia-magoni commented:
you already have https://github.com/freeCodeCamp/freeCodeCamp/pull/61651 open, please update that one, do not create a new PR

rajatbaranwal commented:
Dear Ilenia,

Thank you for the feedback.

I have:

   -

   Removed all the unwanted files from the PR.
   -

   Only added .trim() where it was missing.

The PR is now updated and ready for review.

Thank you for your guidance!

Best regards,
Rajat Baranwal

On Sun, Aug 3, 2025 at 10:56 PM Ilenia ***@***.***> wrote:

> *ilenia-magoni* left a comment (freeCodeCamp/freeCodeCamp#61656)
> <https://github.com/freeCodeCamp/freeCodeCamp/pull/61656#issuecomment-3148585063>
>
> you already have #61651
> <https://github.com/freeCodeCamp/freeCodeCamp/pull/61651> open, please
> update that one, do not create a new PR
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/freeCodeCamp/freeCodeCamp/pull/61656#issuecomment-3148585063>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6FEMJFULQJJEPUX4VEJ6NL3LZA4TAVCNFSM6AAAAACDAGAXB2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTCNBYGU4DKMBWGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
